library/runlevel/src/modules/Service.rb

```diff
@@ -61,19 +61,19 @@ def call(command_name, service_name)
 
       systemd_command =
         case command_name
-        when "show"    then :show
-        when "status"  then :status
-        when "start"   then :start
-        when "stop"    then :stop
-        when "enable"  then :enable
-        when "disable" then :disable
-        when "restart" then :restart
-        when "reload"  then :reload
-        when "try-restart" then :try_restart
-        when "reload-or-restart" then :reload_or_restart
-        when "reload-or-try-restart" then :reload_or_try_restart
-        else
-          raise "Command '#{command_name}' not supported"
+      when "show"    then :show
+      when "status"  then :status
+      when "start"   then :start
+      when "stop"    then :stop
+      when "enable"  then :enable
+      when "disable" then :disable
+      when "restart" then :restart
+      when "reload"  then :reload
+      when "try-restart" then :try_restart
+      when "reload-or-restart" then :reload_or_restart
+      when "reload-or-try-restart" then :reload_or_try_restart
+      else
+        raise "Command '#{command_name}' not supported"
```

Comment:
ugh, no, this is plainly wrong, why elsif is aligned with if, but when not aligned with case? this indentation looks broken to me.